assets/js/Sign.jsx

```diff
@@ -144,6 +144,13 @@ class Sign extends Component {
             <span style={fontSize(signId)}>
               {signId}
             </span>
+            {readOnly
+              && (
+                <div className="viewer--mode-text">
```

Comment:
Are there also changes to a CSS file to add a style for that class? I'm just seeing the changes to this and the accompanying test file.

Reply:
there are no changes i thought were worth keeping.  it does make sense to me to have a class though, regardless of style, but i can remove it if you think thats bad

Reply:
Nah, that's fine.